Makefile.vars.mk

```
@@ -18,7 +18,12 @@ KIND_KUBECONFIG ?= $(TESTBIN_DIR)/kind-kubeconfig-$(KIND_NODE_VERSION)
 KIND_CLUSTER ?= k8up-$(KIND_NODE_VERSION)
 KIND_KUBECTL_ARGS ?= --validate=true
 
-E2E_TAG ?= e2e_$(shell sha1sum $(BIN_FILENAME) | cut -b-8)
+ifeq ($(shell which sha1sum 2>&1; echo $$?),0)
+  SHA1SUM = sha1sum
+else
+  SHA1SUM = shasum -a1
+endif
+E2E_TAG ?= e2e_$(shell $(SHA1SUM) $(BIN_FILENAME) | cut -b-8)
```

Alternatively, this could also be written more concise:

```suggestion
SHASUM ?= $(shell command -v sha1sum > /dev/null && echo "sha1sum" || echo "shasum -a1")
E2E_TAG ?= e2e_$(shell $(SHASUM) $(BIN_FILENAME) | cut -b-8)
```

---

> Alternatively, this could also be written more concise:

I'm open for the most idiomatic and portable way to do it. I'd prefer readability over conciseness. I've seen the approach to try the first command and then the next one if that fails. It works, but also is less expressive.